tests/test_data_gridded.py

```diff
@@ -0,0 +1,200 @@
+
+import os
+import unittest
+from copy import deepcopy
+from glob import glob
+
+import requests
+from inicheck.tools import cast_all_variables
+from inicheck.utilities import pcfg
+
+from smrf.framework.model_framework import can_i_run_smrf, run_smrf
+from tests.test_configurations import SMRFTestCase
+
+
+class TestLoadGrid(SMRFTestCase):
+
+    def compare_hrrr_gold(self, out_dir):
+        """
+        Compare the model results with the gold standard
+
+        Args:
+            out_dir: the output directory for the model run
+        """
+
+        output_dir = os.path.join(self.test_dir, out_dir)
+        s = os.path.join(output_dir, '*.nc')
+        file_names = glob(os.path.realpath(s))
+
+        # path to the gold standard
+        gold_path = os.path.realpath(
+            os.path.join(self.test_dir, 'RME', 'gold_hrrr'))
+
+        for file_name in file_names:
+            nc_name = file_name.split('/')[-1]
```

Comment:
Same as
```
os.path.basename(path)
```
?
Basename is cross platform.

Reply:
So instead of `realpath` use `basename`?

Reply:
instead of `file_name.split`